### PR TITLE
[I18N][17.0] survey: slightly improve translation in certification

### PR DESCRIPTION
**Pull request by daiduongnguyen-odoo**
Part of ticket:
https://viindoo.com/web#id=54378&cids=1&menu_id=89&model=viin.helpdesk.ticket&view_type=form

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/survey/i18n/vi.po [hunk 1]
```diff
@@ -666,7 +666,7 @@ msgstr "<span>Khảo sát đã hoàn thành</span>"
 #. module: survey
 #: model_terms:ir.ui.view,arch_db:survey.certification_report_view_general
 msgid "<span>Date</span>"
-msgstr "<span>Ngày</span>"
+msgstr "<span></span>"
```
Comment (daiduongnguyen-odoo) [inline addons/survey/i18n/vi.po:669]: Nên fix cho odoo ở `https://github.com/odoo/odoo/pull/184422`

**Comment by phamgiang2510** [comment]
@viinbot rebase

**Comment by phamgiang2510** [comment]
@viinbot r+

**Comment by phamgiang2510** [comment]
@viinbot r+

**Pull request merged**